src/lib/components/scroll_area.svelte

```diff
@@ -20,18 +20,14 @@
 	}> = $props();
 
 	let scroll_area = $state<HTMLElement>();
-	let add_mask_bottom = $state<boolean>(true),
-		expanded = false; // we might need this later
-
-	let first_element_intersecting = $state<boolean>(),
-		end_element_intersecting = $state<boolean>(),
-		first_element = $state<HTMLElement>(),
-		end_element = $state<HTMLElement>();
```

Comment:
Wt.. you again removed 70% of codes?
reminds me: https://github.com/baseplate-admin/coreproject-v4-ui/pull/131

Reply:
yes cause i am investigating if this works